kettle-plugins/formats/src/main/resources/org/pentaho/big/data/kettle/plugins/formats/parquet/messages/messages_en_US.properties

```diff
@@ -0,0 +1,9 @@
+BaseStepDialog.StepName=Step name:
```

Comment:
@tgf Sorry. Should have been more specific. I think Ben had some issues with them showing on Linux. What OS do you use?